python/nano/src/bigdl/nano/deps/ipex/ipex_api.py

```diff
@@ -48,16 +48,19 @@ def PytorchIPEXJITModel(model, input_sample=None, use_ipex=False,
     :param thread_num: the thread num allocated for this model.
     :param inplace: whether to perform inplace optimization. Default: ``False``.
     :param jit_strict: Whether recording your mutable container types.
+    :param jit_converter: use ``jit.trace`` or ``jit.script`` to
```

Comment:
In my feeling, `jit_converter` is not intuitive and a little strange.

I think `jit_compiler` may be a choice：
_Scripting a function or nn.Module will inspect the source code, compile it as TorchScript code using the TorchScript compiler, and return a [ScriptModule]_\
_Trace a function and return an executable or [ScriptFunction] that will be optimized using just-in-time compilation._

Reply:
I also think that `jit_converter` could be not so intuitive.

However, after some investigation, I think one of the biggest difference between `jit.script` and `jit.trace` is that `jit.script` will "compiles" a model into graph, but `jit.trace` will not.

From the direct explanation in TorchScript official documentation:
![image](https://user-images.githubusercontent.com/54161268/207514212-08c93abe-04c0-4b1d-9935-7ede12003afc.png)
From my understanding, it also shows that the complication of model converted from `jit.trace` will be conducted **jit in time** (but the complication will not be done by `jit.trace` itself).

So, for me, I feel like that `jit_compiler` will by itself already refer to `jit.script`, so may not be a suitable parameter name.

@TheaperDeng Do you have any idea what will be a more suitable name?

Reply:
maybe we could just call it `jit_method` after some discussion with @MeouSker77 @rnwang04

Reply:
Have renamed the parameter.